cmd/utils/flags.go

```diff
@@ -804,12 +804,50 @@ var (
 		Value:    "",
 		Category: flags.BuilderCategory,
 	}
+
+	// Builder rate limit settings
+
+	// BuilderRateLimitDuration determines rate limit of events processed by builder.
```

Comment:
Shouldn't this comment be omitted since it's already in the `Usage`?

Reply:
good point, updated

Reply:
I see this comment is still present both as a comment and in the flag description